travis.sh

```diff
@@ -0,0 +1,243 @@
+#!/bin/bash
+
+## This is a "common" script that can be run on travis CI at a downstream github repository.
+## 
+## Input
+##  .travis.rosintall: A .rosintall file that's used to download the resource of the dependended packages that need to be built from source.
+## 
+## Greatly inspired by JSK travis with additional comments for maintainability: https://github.com/jsk-ros-pkg/jsk_travis 
+## Author: Isaac I. Y. Saito (TORK)
```

Comment:
Not an answer but if we use BSD, then we might as well [add license and copyrights header per file](http://stackoverflow.com/a/3053725/577001).